rs/ic_os/config_types/src/lib.rs

```diff
@@ -176,6 +183,24 @@ const fn default_vm_nr_of_vcpus() -> u32 {
     64
 }
 
+/// HostOS development configuration. These settings are strictly used for development images.
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct HostOSDevSettings {
+    pub vm_memory: u32,
+    pub vm_cpu: String,
+    pub vm_nr_of_vcpus: u32,
```

Comment:
I haven't used `_override` for these fields now that they live in `HostOSDevSettings`. Is this clear enough with the doc comments?